src/Qualys/functions/public/Get-QualysScanSchedules.ps1

```diff
@@ -0,0 +1,48 @@
+﻿<#
+.Synopsis
+    List schedules for vulnerability scans
+.DESCRIPTION
+    List schedules for vulnerability scans
+.PARAMETER ID
+    The ID of the scan schedule you want to display.
+.PARAMETER Active
+    Specify for active schedules only
+.PARAMETER Deactivated
+    Specify for deactivated schedules only
+.EXAMPLE
+    Get-QualysScanSchedules -Active
+#>
+function Get-QualysScanSchedules{
+    [CmdletBinding()]
+    param (
+        [String]$ID,
+        [Switch]$Active,
+        [Switch]$Deactivated
+    )
+
+    process{
+
+        $RestSplat = @{
+            Method = 'GET'
+            RelativeURI = 'schedule/scan/'
+            Body = @{
+                action = 'list'
+                echo_request = '1'
+                show_notifications = '1'
+            }
+        }
+
+        If($Active){
+            $RestSplat.Body['active'] = '1'
```

Comment:
We could raise a courtesy error here is someone specifies both 'active' and 'deactivated'.

Reply:
Good catch, I'd recommend making use of parameter sets to prevent that from being possible

Reply:
Will use a parameter set in next release to avoid this